src/main/scala/nl/knaw/dans/easy/fsrdb/FsRdbUpdater.scala

```diff
@@ -67,7 +92,7 @@ object FsRdbUpdater {
     XML.load(FedoraClient.getObjectXML(pid).execute().getEntityInputStream)
   }
 
-  private def getFileItem(pid: String)(objectXML: Elem)(implicit s: Settings): Try[FileItem] = Try {
+  private def getFileItem(datasetPid: String)(pid: String)(objectXML: Elem)(implicit s: Settings): Try[FileItem] = Try {
```

Comment:
Rename `pid` to `filePid` for clarity?

The [currying of parameters](http://www.codecommit.com/blog/scala/function-currying-in-scala) is only necessary for `s` I think.